features/jmx-config-generator/src/test/resources/cassandra21x-datacollection-config.xml

```diff
@@ -0,0 +1,502 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<jmx-datacollection-config xmlns="http://xmlns.opennms.org/xsd/config/jmx-datacollection">
+    <jmx-collection name="JSR160-anyservice">
+        <rrd step="300">
+            <rra>RRA:AVERAGE:0.5:1:2016</rra>
+            <rra>RRA:AVERAGE:0.5:12:1488</rra>
+            <rra>RRA:AVERAGE:0.5:288:366</rra>
+            <rra>RRA:MAX:0.5:288:366</rra>
+            <rra>RRA:MIN:0.5:288:366</rra>
+        </rrd>
+        <mbeans>
+            <mbean name="java.lang.GarbageCollector" objectname="java.lang:type=GarbageCollector,name=ConcurrentMarkSweep">
+                <attrib name="CollectionCount" alias="0CollectionCnt" type="gauge"/>
+                <attrib name="CollectionCount" alias="1CollectionCnt" type="gauge"/>
```

Comment:
Like mentioned before this config seems to be broken because of the duplicate attributes.